src/components/Admin/ShowcaseCard.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <section class="text-center">
+    <q-file class="hidden" multiple ref="artsFileRef" v-model="modelArts" @update:model-value="addArts" />
+    <q-btn flat icon="add_circle_outline" label="Upload Arts" rounded @click="onUploadArts" />
```

Comment:
```suggestion
    <q-btn flat icon="add_circle_outline" label="Upload Art" rounded @click="onUploadArts" />
```

Technically, "art" is not pluralized as "arts" -- why? I do not know. English makes no sense. I've changed the Label but NOT changed any of the other function calls. There were 22 "Arts" in the actual code (including Props and Events) so I was hesitant to make this change to broadly. Besides -- no one will see the code but us so "Arts" is fine in the code base.

Reply:
Really?! I would never know that.